lib/python/ray/worker.py

```diff
@@ -775,6 +775,17 @@ def put(value, worker=global_worker):
   worker.put_object(objectid, value)
   return objectid
 
+def select(objectids, num_objects=0, worker=global_worker):
+  check_connected(worker)
+  ready_ids = raylib.is_ready(worker.handle, objectids)
+  while sum(ready_ids) < num_objects:
+    ready_ids = raylib.is_ready(worker.handle, objectids)
+  result = [index for index, _ in enumerate(objectids) if ready_ids[index]] 
+  if (num_objects != 0):
+     return result[0:num_objects]
+  else: return result
```

Comment:
I wanted to avoid looping, but it turns out the implementation in that case is pretty tricky. This is good for now, but we should probably include a `sleep` in the loop to avoid sending too many requests.